spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -173,6 +173,7 @@ def in_person_deadline_passed
   def in_person_ready_to_verify
     UserMailer.with(user: user, email_address: email_address_record).in_person_ready_to_verify(
       enrollment: in_person_enrollment,
+      is_enhanced_ipp: false,
     )
   end
```

Comment:
WDYT of having a second preview here to show what it looks wiht with EIPP enabled?

```suggestion
  def in_person_ready_to_verify
    UserMailer.with(user: user, email_address: email_address_record).in_person_ready_to_verify(
      enrollment: in_person_enrollment,
      is_enhanced_ipp: false,
    )
  end
  
  def in_person_ready_to_verify_enhanced_ipp_enabled
    UserMailer.with(user: user, email_address: email_address_record).in_person_ready_to_verify(
      enrollment: in_person_enrollment,
      is_enhanced_ipp: true,
    )
  end
```

Reply:
(doing this would simplify the test instructions to not need to modify the code to switch between versions of the email)

Reply:
Yes! Thanks for flagging this. I think I started to do this and ran into a linter error and decided I'd come back. (Either I did it in the wrong file or named it the same.) I did not come back to it so thank you for calling it out.  
See commit 79d63669238af115277f4afab1563217c8f3acab